router/routes.js

```diff
@@ -2,36 +2,48 @@ require('dotenv').config();
 const mongoose = require('mongoose');
 const express = require('express');
 const questionnaireModel = require('../models/Mongoschema');
-const createQuestionnaire = require('../controller/questionnairecontroller');
+const questionnaireController = require('../controller/questionnairecontroller');
 
 const router = express.Router();
 
-const uri = process.env.DB_URI;
-
 // GET /health --> returns database status
 router.get('/health', function (req, res) {
-  try {
-    mongoose.connect(uri); //TODO: ask for connection status ¿await mongoose.authenticate().then(() =>?
-    res.status(200).set('Content-Type', 'application/json').send({ response: 'Connection has been established successfully.' });
-  } catch (error) {
-    res.status(500).set('Content-Type', 'application/json').send('Unable to connect to the database:');
-  }
+  const dbState = mongoose.STATES[mongoose.connection.readyState];
+    res.set('Content-Type', 'application/json').send({ response: dbState });
 });
 
 // POST /questionnaire --> receives the questionnaire and stores it in the database
 router.post('/questionnaire', async (req, res) => {
   try {
     const questionnaireParams = req.body;
-    const questionnaire = await createQuestionnaire({
+    const questionnaire = await questionnaireController.createQuestionnaire({
       questionnaireParams,
       questionnaireModel,
     });
     res.status(200).set('Content-Type', 'application/json').send(questionnaire);
   } catch (error) {
-    res.status(500).set('Content-Type', 'application/json').send(error);
+    res.status(400).set('Content-Type', 'application/json').send(error);
   }
 });
 
-module.exports = router;
-
 // GET /questionnaire/id: --> returns the questionnaire
+router.get('/questionnaire/:id', async (req, res) => {
+  try {
+    console.log(req.params, "params");
```

Comment:
`console.log` is for debugging purpose. Please remove it. We will implement `logs` in a different why in order to follow the flow a request